_includes/neurips/2020/about.neurips.html

```diff
@@ -0,0 +1,515 @@
+                <tr>
+                    <td class="tg-h2tm"><span style="font-weight:normal;background-color:#D5A6BD">05:00 PM</span></td>
+                    <td class="tg-pfmh" colspan="2"><span style="font-weight:normal;color:#45494E;background-color:#EA9999">Poster Session</span></td>
+                </tr>
+                <tr>
+                  <td class="tg-h2tm"><span style="font-weight:normal;background-color:#D5A6BD">07:00 PM</span></td>
+                  <td class="tg-0y8e" colspan="3"><span style="background-color:#FFF2CC">Break 3</span></td>
+              </tr>
+                <tr>
+                    <td class="tg-h2tm"><span style="font-weight:normal;background-color:#D5A6BD">07:15 PM</span></td>
+                    <td class="tg-pfmh" colspan="3"><span style="font-weight:normal;color:#45494E;background-color:#EA9999">Social Gathering</span></td>
+                </tr>
+                </tbody>
+            </table>
+        </div>
+        <a class="btn_icml20-blue" href="/neurips-2020-schedule">Detailed Schedule</a>
+    </div> -->
+
+
+
+<!--    <br/><hr> -->
+    <!-- <section id="speakers" >
+      <div class="container">
+        <div class="row">
+          <div class="col-lg-12 text-center">
+            <h2 class="section-heading">Keynote Speakers</h2>
+            <h3 class="section-subheading text-muted"></h3>
+          </div>
+        </div>
+        <div class="grid">
+          {% assign featuredItems = site.tags.keynote-neurips2020  %}
+          {% for post in featuredItems %}
+          <div class="cell speakers-item">  -->
+
+<!--            <a href="{{post.website}}" target="_blank" class="speakers-link">-->
+<!--              <div class="speakers-hover">-->
+<!--&lt;!&ndash;                <div class="speakers-hover-content">&ndash;&gt;-->
+<!--&lt;!&ndash;                  <p>{{post.topic}}</p>&ndash;&gt;-->
+<!--&lt;!&ndash;                  <i class="fa fa-plus fa-3x"></i>&ndash;&gt;-->
+<!--&lt;!&ndash;                </div>&ndash;&gt;-->
+<!--              </div>-->
+<!--&lt;!&ndash;              <img src="/assets/img/icml/2020/presenters/{{ post.img }}" class="responsive-image img-centered desaturate" alt="">&ndash;&gt;-->
+<!--            </a>-->
+
+
+<!--              <a href="{{post.website}}" target="_blank" >
+                <img src="/assets/img/neurips/2020/presenters/{{ post.img }}" class="responsive-image img-centered" alt="">
+              </a> -->
+
+
+
+
+<!--            <div class="speakers-caption">-->
+<!--              <h4>{{ post.name }}</h4>-->
+<!--              <p class="text-muted">{{ post.title }}</p>-->
+<!--            </div>-->
+<!--          </div>
+          {% endfor %}
+        </div>
+      </div>
+    </section>-->
+
+
+
+    <hr>
+    <div  id="workshop-org">
+
+      <h2 style="text-align: center">Organizers</h2>
+
+      <h3>Workshop Chairs</h3>
+      <ul>
+        <li><em>General & Logistics Co-Chairs --</em> Maria Luisa Santiago (Accel AI)</li>
```

Comment:
No General and Logistics Co-Chairs should be listed - Maria is fulfilling this role as part of her employment with the Accel AI Institute.